config/webpackDevServer.config.js

```diff
@@ -9,6 +9,7 @@ const paths = require("./paths");
 const protocol = process.env.HTTPS === "true" ? "https" : "http";
 const host = process.env.HOST || "0.0.0.0";
 const serverPort = process.env.PORT || 3000;
```

Comment:
At some point, we need to harmonize what the client uses to read config with what the server is using.

Reply:
Yes we should, step by step :-)